stdlib/result.mli

```diff
@@ -39,11 +39,23 @@ val get_ok : ('a, 'e) result -> 'a
 
     @raise Invalid_argument if [r] is [Error _]. *)
 
+val get_ok' : ('a, string) result -> 'a
+(** [get_ok'] is like {!get_ok} but in case of error uses the
+    error message for raising [Invalid_argument].
+
+    @since 5.4 *)
+
 val get_error : ('a, 'e) result -> 'e
 (** [get_error r] is [e] if [r] is [Error e] and raise otherwise.
 
     @raise Invalid_argument if [r] is [Ok _]. *)
 
+val error_to_failure : ('a, string) result -> 'a
+(** [error_to_failure r] is [v] if [r] is [Ok v] and raises [Failure e]
+    if [r] is [Error e].
+
+    @since 5.4 *)
```

Comment:
Are `@raise` annotations needed here and above?

Reply:
For some reason the build didn't like them (no reason specified the doc build failed) I left them out. I think the doc strings are clear enough.